clowder-framework/clowder2
# 1061 filesystem UI component

To test:
Run an extractor, preferably locally to do the necessary modifications:
  1. Add a parameter in `extractor-info.json` file,  in the `schema` field of  `parameters` 
  ```json
      "DATASET_FILE_ID":{
              "type":"string",
              "title":"Dataset file ID",
              "description":"The Clowder file id of the dataset file to be used for inference",
              "format": "clowderFile"
         },
  ```
  2. The extractor should have a field like this
  ![image](https://github.com/clowder-framework/clowder2/assets/16596733/32266bc3-111d-47d5-8e75-4187da206448)
